mcode/mcode-extraction-framework
# Generic valueSet lookup

# Summary
Implements a generic valueSet helper for loading valueSets from provided paths, replacing the numerous implementations of checkCodeInVs with a single one. Additionally, this task unearthed an issue with the mCODE condition valueSets. After raising this with Mark and May, they pushed out an updated version of valueSets with ICD-10 codes in them, now using the period separator. This PR incorporates those new valueSets.

## New behavior
Anywhere that implemented a checkCodeInVs function now uses the one defined by `valueSetUtils.js`. This file also defines an enum of possible valueSet file types supported; currently only the JSON type has a defined loader and parser. 

## Code changes
- Major changes to fixtures on the tails of the new ValueSets

# Testing guidance
- Ensure that the new tests for `valueSetUtils.js` are sufficient
- Ensure that all tests pass
- Ensure that everything runs as expected with the CLI
